eagle-jpm/eagle-jpm-util/src/main/java/org/apache/eagle/jpm/util/jobrecover/RunningJobManager.java

```diff
@@ -49,12 +52,14 @@ private CuratorFramework newCurator(String zkQuorum, int zkSessionTimeoutMs, int
         );
     }
 
-    public RunningJobManager(String zkQuorum, int zkSessionTimeoutMs, int zkRetryTimes, int zkRetryInterval, String zkRoot) {
+    public RunningJobManager(String zkQuorum, int zkSessionTimeoutMs, int zkRetryTimes, int zkRetryInterval, String zkRoot, String lockPath) {
         this.zkRoot = zkRoot;
+        curator = newCurator(zkQuorum, zkSessionTimeoutMs, zkRetryTimes, zkRetryInterval);
+        curator.start();
```

Comment:
In the future,we should follow http://stackoverflow.com/questions/20384869/when-curatorframeworks-methods-start-and-close-shoud-be-called